app/controllers/films_controller.rb

```diff
@@ -46,6 +46,12 @@ def destroy
     end
   end
 
+  def delete_film_link_attachment
+    @film_link = ActiveStorage::Attachment.find(params[:id])
```

Comment:
This business logic shouldn't be in a controller. Try to move it to for instance `Service`
Also it will be more DRY because you can reuse this service in other places

Reply:
At the moment I only need  a possibility to delete big attachment for better storage management. If that can be reused later I would move to the Service...

Reply:
Reusability is only a benefit, try to keep your controllers clean